README.md

```diff
@@ -332,6 +337,40 @@ Developed in Mexico by [Conekta](https://www.conekta.com). Available with [MIT L
 
 ***
 
+## How to contribute to the project
+
+1. Fork the repository
+ 
+2. Clone the repository
+```
+    git clone git@github.com:yourUserName/conekta-php.git
+```
+3. Create a branch
+```
+    git checkout develop
+    git pull origin develop
+    # You should choose the name of your branch
+    git checkout -b <feature/my_branch>
+```    
+4. Make necessary changes and commit those changes
+```
+    git add .
+    git commit -m "my changes"
+```
+5. Push changes to GitHub
+```
+    git push origin <feature/my_branch>
+```
+6. Submit your changes for review, create a pull request
+
+   To create a pull request, you need to have made your code changes on a separate branch. This branch should be named like this: **feature/my_feature** or **fix/my_fix**.
+
+   Make sure that, if you add new features to our library, be sure to add the corresponding **unit tests**.
```

Comment:
Podríamos cambiarlo a "be sure that corresponding **unit tests** are added"?

Reply:
ok